lib/web_ui/dev/felt.dart

```diff
@@ -47,12 +47,17 @@ void main(List<String> args) async {
   } on ToolException catch (e) {
     io.stderr.writeln(e.message);
     exitCode = 1;
+  } on ProcessException catch (e) {
+    io.stderr.writeln('description: ${e.description}'
+        'executable: ${e.executable} ${e.arguments} '
+        'exit code: ${e.exitCode}');
```

Comment:
Does it make sense to print each one on its own line?

Reply:
done.